BlockSettleUILib/BSTerminalMainWindow.cpp

```diff
@@ -1362,24 +1326,9 @@ void BSTerminalMainWindow::setupMenu()
 void BSTerminalMainWindow::openAuthManagerDialog()
 {
    allowAuthAddressDialogShow_ = false;
-   openAuthDlgVerify(QString());
-}
-
-void BSTerminalMainWindow::openAuthDlgVerify(const QString &addrToVerify)
-{
-   const auto showAuthDlg = [this, addrToVerify] {
-      authAddrDlg_->setModal(true);
-      authAddrDlg_->show();
-      QApplication::processEvents();
-      authAddrDlg_->setAddressToVerify(addrToVerify);
-   };
-   if (authManager_->HaveAuthWallet()) {
-      showAuthDlg();
-   } else {
-      createAuthWallet([this, showAuthDlg] {
-         QMetaObject::invokeMethod(this, showAuthDlg);
-      });
-   }
+   authAddrDlg_->setModal(true);
+   authAddrDlg_->show();
```

Comment:
Could be just ->exec() instead of these 2 calls?

Reply:
It can. Done.